Dockerfile

```diff
@@ -8,5 +8,7 @@ FROM alpine:3 AS release
 WORKDIR /
 COPY --from=builder /app/bin/main /main
 
+COPY .env /.env
```

Comment:
Do not add env file to image, as this will result in image file that will be preconfigured. 
Better approach is to use "environment" directive in docker compose instead.

Reply:
I did this so the go application in the container can read the configuration from this .env file.